src/test/java/org/opensrp/web/rest/OrganizationResourceTest.java

```diff
@@ -485,8 +485,6 @@ public void testGetUserAssignedLocationsAndPlansWithPlansShouldReturnActivePlans
 				.collect(Collectors.toSet());
 
 		assertEquals(activePlans.size(), userAssignment.getPlans().size());
-
-		assertTrue(activePlans.size() < planIds.size());
```

Comment:
Reason for removing this assert?

Reply:
this https://github.com/opensrp/opensrp-server-web/blob/7439152a35e35ad81c1d01638ac725b39db18928/src/test/java/org/opensrp/web/rest/OrganizationResourceTest.java#L533